src/mach/pack/alpha/impl/tools_deps.clj

```diff
@@ -55,10 +55,7 @@
 
 (defn system-edn
   []
-  (-> "mach/pack/alpha/system_deps.edn"
-      io/resource
-      slurp
-      edn/read-string))
+  (tools.deps.reader/read-deps (tools.deps.reader/default-deps)))
```

Comment:
I don't want to use the default-deps here, I don't want `:deps` from the user's `~/.clojure/deps.edn` to be factored into creating uberjars. This should be the install-deps instead.

Reply:
I've changed this argument to nil as read-deps merges it's arguments into (install-deps) anyway.

Reply:
This has been updated again (system-edn is no longer needed as install-deps can be inlined)